src/BestBooks.js

```diff
@@ -1,30 +1,89 @@
 import React from 'react';
+import axios from 'axios';
+import Carousel from 'react-bootstrap/Carousel';
+
+const SERVER = process.env.REACT_APP_SERVER;
 
 class BestBooks extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      books: []
+      books: '',
+    };
+  }
+
+  /* DONE: Make a GET request to your API to fetch books for the logged in user  */
+  async componentDidMount() {
+    let response = await axios.get(`${SERVER}/books`);
+    console.log(response.data);
+    // set response from axios to state
+    try {
+      this.setState({ books: response.data });
+    } catch (error) {
+      console.log(error);
     }
   }
 
-  /* TODO: Make a GET request to your API to fetch books for the logged in user  */
+  // add this.state.books to render return
 
   render() {
-
     /* TODO: render user's books in a Carousel */
 
     return (
       <>
         <h2>My Essential Lifelong Learning &amp; Formation Shelf</h2>
 
-        {this.state.books.length ? (
-          <p>Book Carousel coming soon</p>
+        {this.state.books ? (
```

Comment:
- Review demo for this line here :) I believe you can also use length of array for this